src/main/java/org/rsa/aws/RequestsManager.java

```diff
@@ -0,0 +1,61 @@
+package org.rsa.aws;
+
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
+import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
+import software.amazon.awssdk.enhanced.dynamodb.model.*;
+
+import java.util.LinkedList;
+import java.util.Optional;
+import java.util.Queue;
+
+
+public class RequestsManager<T> {
+    private final Queue<UpdateItemEnhancedRequest<T>> _updateRequestsQueue = new LinkedList<>();
+    private final DynamoDbTable<T> _table;
+    private final Class<T> _classType;
+    private boolean _processingUpdateRequests = false;
```

Comment:
What's with the `_`? This is commonly used to describe a private variable where it would otherwise be public. Because Jave provides the functionality of public/private variables, we shouldn't be prefixing the names with `_`.